[docs] Replace some images, gifs with videos (#2662)
mui/mui-toolpad: docs/data/toolpad/concepts/building-ui.md

## Component Library

> In order to access a list of built in components, hover over the **Component library** vertical bar to make it expand.

Reviewer: ```suggestion
To access the list of available Toolpad components, hover your cursor over the vertical bar labeled **Component library** to expand it.
```

<video controls width="auto" height="100%" style="contain" alt="component-library">
  <source src="/static/toolpad/docs/concepts/building-ui/component-library.mp4" type="video/mp4">
  Your browser does not support the video tag.
</video>

---

> The list of built-in components available in Toolpad can be found [here](/toolpad/reference/components/) and we are always adding more. To make it easier for us to understand what should be added next, please help by [upvoting](https://github.com/mui/mui-toolpad/labels/waiting%20for%20%F0%9F%91%8D) components.

Reviewer: I trued to use more direct language instead of "can be found", "should be added", "by upvoting"... It feels more [conversational](https://www.notion.so/mui-org/Writing-style-guide-2a957a4168a54d47b14bae026d06a24b?pvs=4#8852c4ad7c994c06b07358bbc2885188).

```suggestion
Toolpad has many [built-in components](/toolpad/reference/components/) and we are always adding more. To make it easier for us to understand what to add next, please [upvote](https://github.com/mui/mui-toolpad/labels/waiting%20for%20%F0%9F%91%8D) the components you need, or open a [feature request](https://github.com/mui/mui-toolpad/issues/new?assignees=&labels=status%3A+needs+triage&projects=&template=2.feature.yml).
```

## Using components

1. Open the component library and drag a component over the canvas.
2. The canvas will highlight the location where the component can be dropped. Release the mouse to drop it.

> 3. To select a component, click on it either in the canvas or in the page hierarchy explorer. An overlay indicates the selected component.

Reviewer: Would it make sense to link the word "page hierarchy explorer" to a doc describing what it is and how to find it?

> 3. To select a component in canvas, click either from the canvas or the page heirarchy explorer. An overlay indicates the selected component.

Reviewer 2: ```suggestion
3. To select a component in canvas, click on it either in the canvas or in the page hierarchy explorer. An overlay indicates the selected component.
```

> 4. To move a component, drag it from the drag handler present in the overlay and drop it at its new location. The canvas will highlight available drop locations with a blue indicator.

Reviewer: ```suggestion
4. To move a component, drag it from its drag handle in the overlay and drop it at its new location. The canvas will highlight available drop locations with a blue indicator.
```

> 4. To reposition a component, click on the drag handler present in the overlay, drag and drop it at its new location. The canvas will highlight available drop locations with a blue indicator.

Reviewer 2: ```suggestion
4. To move a component, drag it from the drag handler present in the overlay and drop it at its new location. The canvas will highlight available drop locations with a blue indicator.
```

> 5. To delete a component, click the bin icon in the overlay to remove it from the canvas. You can also press the <kbd class="key">Backspace</kbd> key.

Reviewer 2: ```suggestion
5. To delete a component, click the bin icon in the overlay. You can also press the <kbd class="key">Backspace</kbd> key while the component is selected.
```

6. Components can be horizontally resized within the grid boundaries:

{{"component": "modules/components/DocsImage.tsx", "src": "/static/toolpad/docs/concepts/building-ui/canvas-8.png", "alt": "Resize component", "caption": "Resizing components along columns", "zoom": false }}

> Each component has a set of properties that you can tweak from the link button in the **inspector panel** on the right. The **page hierarchy explorer** on the bottom-left can be used to view the page structure, and select or rename components.

Reviewer: ```suggestion
Each component has a set of properties that you can tweak in the **inspector panel** on the right. To view the page structure, and select or rename components, use the **page hierarchy explorer** on the bottom-left.
```

> Each component has a set of properties that you can tweak from the link button in the **inspector panel** on the right. The **page hierarchy explorer** on the bottom-left can be used for viewing page structure, component selection and renaming.

Reviewer 2: ```suggestion
Each component has a set of properties that you can tweak from the link button in the **inspector panel** on the right. The **page hierarchy explorer** on the bottom-left can be used to view the page structure, and selecting or renaming components.
```
